entities.py

```diff
@@ -43,7 +44,7 @@ class Actor(Entity):
     An actor is an entity that can perform actions.
     """
 
-    def __init__(self, name, pos, behavior):
+    def __init__(self, name, pos, behavior, char, color):
```

Comment:
Shouldn't entities have char and color attributes too? The way I understand it, an entity is anything that has a physical representation in the world.

Reply:
You are technically correct, the best kind of correct!